app/models/health_care_application.rb

```diff
@@ -302,15 +301,13 @@ def log_submission_failure_details
       error_class: 'HealthCareApplication FailedWontRetry'
     )
 
-    log_message_to_sentry(
-      'HCA total failure',
-      :error,
+    Rails.logger.info(
```

Comment:
Out of curiosity: how come you've made this log level `info` vs keeping it as `error`